runtime/android/core/src/org/xwalk/core/XWalkCoreWrapper.java

```diff
@@ -561,6 +518,17 @@ public Object getWrapperObject(Object object) {
         return null;
     }
 
+    private String getApplicationMetaData(String name) {
```

Comment:
`XWalkCoreWrapper` is in the lower level. Although the code is almost the same, but it's bundle with the context which is the private member